test/run-local.sh

```diff
@@ -20,7 +25,7 @@ until curl --output /dev/null --silent --fail --insecure https://127.0.0.1:5556/
 done
 
 (
-  ./observatorium \
+  ${OBSERVATORIUM} \
```

Comment:
nit. you don't need curly braces here if you don't use any of its fancy features, e.g. default if empty, etc.